app_dart/lib/src/request_handlers/postsubmit_luci_subscription.dart

```diff
@@ -82,9 +83,13 @@ class PostsubmitLuciSubscription extends SubscriptionHandler {
     }
 
     final bbv2.Build build = buildsPubSub.build;
+
     // Note that result is no longer present in the output.
     log.fine('Updating buildId=${build.id} for result=${build.status}');
 
+    // Add build fields that are stored in a separate compressed buffer.
+    build.mergeFromBuffer(ZLibCodec().decode(buildsPubSub.buildLargeFields));
```

Comment:
Have we been building the wrong commits post-submit too?!

Reply:
Although, I suppose the git_ref field is only used in presubmit. Still, I wonder if there are any other correctness issues we might have with prod binaries....

Reply:
I don't know if there have been any specific problems affecting postsubmit.  But IIUC any consumer of `BuildsV2PubSub` should be merging the data from `build_large_fields`.